src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -916,6 +920,7 @@ $divider-color: #000; // Color of dividers between assays with targets
             width: $donor-cell-width;
             padding: 0;
             border-left: none;
+            text-align: right;
```

Comment:
I can see that this line works and that the labels aren't aligned properly without it, but I'm confused because the matrix pages used to not require it, so I think I must have added some line that changes the default, but I can't figure out where. I just think this shouldn't be necessary, but it clearly is. Do you see what the change is that I made that necessitates this update?

Reply:
I will in the middle of something important so will update this answer later. It had to do with display:flex, overriding the style

Reply:
Thank you for looking into this! This seems good!